### PR TITLE
RELATED: PSDK-163 update README.md

### PR DESCRIPTION
**Pull request by hkad98**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: README.md [hunk 1]
```diff
@@ -20,20 +20,23 @@ The [gooddata-sdk](./gooddata-sdk) package provides a clean and convenient Pytho
...
 create pandas series and data frames.
 
 Check out the GoodData Pandas [documentation](https://gooddata-pandas.readthedocs.io/en/latest/) to learn more and get started.
 
+### GoodData Foreign Data Wrapper
+
+[!WARNING]
+GoodData Foreign Data Wrapper is currently deprecated.
+
+The [gooddata-fdw](./gooddata-fdw) package provides a way to map GoodData.CN semantic layer and/or insights stored in your GoodData.CN
```
Comment (Mara3l) [inline README.md:35]: Please remove the .cn

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by Mara3l — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR397

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/211/ : FAILURE in 1m 29s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/361/ : SUCCESS in 1m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/364/ : SUCCESS in 9s

**Pull request merged**